tap_iterable/iterable.py

```diff
@@ -108,9 +109,12 @@ def list_users(self, column_name=None, bookmark=None):
 
 
   def campaigns(self, column_name=None, bookmark=None):
+    bookmark = utils.strptime_with_tz(bookmark)
     res = self.get("campaigns")
     for c in res["campaigns"]:
```

Comment:
please refrain from using single letter variables
```suggestion
    for campaign in res["campaigns"]:
```